sdk/sdk-e2e/tests/refinanceMakerSpark.test.ts

```diff
@@ -133,6 +133,8 @@ describe.skip('Refinance Maker Spark | SDK', () => {
       await sdk.simulator.refinance.simulateRefinancePosition({
         position: makerPosition,
         targetPool: poolAgain,
+        targetCollateral: makerPosition.collateralAmount.token,
```

Comment:
We need to discuss this. The original idea for `targetPool` was that it included the actual collateral and debt that was going to be used. So, for example, in AAVE, `targetPool` would not be the actual AAVE pool, but an abstraction that indicates what type of tokens you are going to use. I think we got sidetracked by adding the collateral and debt config, but I'm not so sure anymore about those, particularly because there is no use for having all the config in the pool. It feels to me that we have 2 different things we want to know:
  - The target pool of a Summer strategy, that should be only the actual token pair (for a lending pool) that we need
  - The supported collaterals/debts pairs on a protocol like AAVE, to be able to choose the right pool
  
  Let's have a discussion when you have time

Reply:
Refactored to sourcePosition -> targetPosition